api-lib/gql/__generated__/zeus/const.ts

```diff
@@ -1662,6 +1662,7 @@ export const AllTypesProps: Record<string, any> = {
     circle_id: 'bigint_comparison_exp',
     created_at: 'timestamptz_comparison_exp',
     id: 'bigint_comparison_exp',
+    server_channel: 'String_comparison_exp',
```

Comment:
Can we call this discord_server_channel instead? otherwise it is pretty ambiguous

Reply:
I'd suggest `discord_channel_id` and `discord_role_id` as that's the nomenclature they use in discord